### PR TITLE
Include GitHub templates

### PR DESCRIPTION
**Pull request by jeanbez**
Include GitHub templates for issues and PRs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+**Bug Report**
+
+A clear and concise description of what the bug is.
```
Comment (zhangwei217245) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What is the minimum set of factors in the description that can make it 'clear and concise'?

**Review by zhangwei217245 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+**To Reproduce**
+
+How are you building/running PDC?
+
+```bash
+...
+```
+
+What is the running configuration file you use?
```
Comment (houjun) [inline .github/ISSUE_TEMPLATE/bug_report.md]: currently we don't need a configuration file, maybe ask for environment variables before we switch to config file

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+...
+```
+
+**Expected Behavior**
+
+A clear and concise description of what you expected to happen.
+
+**Software Environment**
+
+- version of PDC: [e.g. 0.3]
```
Comment (houjun) [inline .github/ISSUE_TEMPLATE/bug_report.md]: ask for branch and commit hash is another option

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jeanbez) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Depends on the case... that's why it is stated as it is. Any suggestions you believe are a must for every case?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (jeanbez) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Changed to running setup

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (jeanbez) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done!

**Review by zhangwei217245 — APPROVED** [review]

**Review by houjun — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+**Bug Report**
+
+A clear and concise description of what the bug is, including module and feature.
+
+**To Reproduce**
+
+How are you building/running PDC?
+
+- version of PDC: [e.g. 0.3, branch, or hash]
```
Comment (sbyna) [inline .github/ISSUE_TEMPLATE/bug_report.md]: comma after e.g.,

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- installed PDC using: [spack, from source]
+- operating system: [name and version]
+- machine: [Are you running on a supercomputer or public cluster?]
+- version of Mercury: [e.g. 1.12.0]
```
Comment (sbyna) [inline .github/ISSUE_TEMPLATE/bug_report.md]: comma after e.g.,

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- name and version of MPI: [e.g. OpenMPI 4.1.1]
```
Comment (sbyna) [inline .github/ISSUE_TEMPLATE/bug_report.md]: e.g.,

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+**Is your feature request related to a problem? If so, please describe.**
```
Comment (sbyna) [inline .github/ISSUE_TEMPLATE/feature_request.md]: How about converting this question into: 
"What does this feature solve or improve?"

**Pull request merged**